docs/changelogs/CHANGELOG-2.28.md

```diff
@@ -0,0 +1,244 @@
+- Kube-state-metrics chart is now using the upstream helm chart. This means there are some adjustements required to the `values.yaml`. The name needs to be overriden. 
+    ```yaml
+    kube-state-metrics:
+    fullnameOverride: kube-state-metrics
+    ```
+### New Features
+
+- LocalAI was added to the default application catalog for the enterprise edition ([#14700](https://github.com/kubermatic/kubermatic/pull/14700))
+- The default apps from the enterprise catalog are now also part of the kubermatic-installer `mirror-images` command ([#14683](https://github.com/kubermatic/kubermatic/pull/14683))
+- Add Kyverno images to `mirror-images` command ([#14681](https://github.com/kubermatic/kubermatic/pull/14681))
+- Add `--insecure` flag to mirror-images command to bypass TLS verification ([#14635](https://github.com/kubermatic/kubermatic/pull/14635))
+- Add support for configuring a Authorization Webhook for UserClusters ([#13930](https://github.com/kubermatic/kubermatic/pull/13930))
+- Mla-secrets: chart can now be safely redeployed, existing secrets take precedence over provided values ([#14568](https://github.com/kubermatic/kubermatic/pull/14568))
+- Gateway API CRDs are now automatically installed in user clusters when KubeLB integration is enabled ([#14368](https://github.com/kubermatic/kubermatic/pull/14368))
+- Add support for scheduling options (tolerations, affinity, nodeSelector) to the KKP operator Helm chart, allowing users to control pod placement in their clusters ([#14574](https://github.com/kubermatic/kubermatic/pull/14574))
+- Add nginx config to increase header size for Dashboard and Dex ([#14579](https://github.com/kubermatic/kubermatic/pull/14579))
+- The default Policy Template Catalog can be deployed via --deploy-default-policy-template-catalog flag (EE) ([#14472](https://github.com/kubermatic/kubermatic/pull/14472))
+- A new --limit-apps flag has been added to the Kubermatic Installer, allowing users to limit which AppDefinitions are installed during the setup process. This flag accepts a comma-separated list of AppDefinition names. If the flag is not provided or the list is empty, all available AppDefinitions will be installed—provided the default app catalog is enabled ([#14569](https://github.com/kubermatic/kubermatic/pull/14569))
+- KubeLB: new field `extraArgs` has been introduced for KubeLB at the seed and cluster level. This field can be used to configure extra arguments for the kubeLB CCM ([#14564](https://github.com/kubermatic/kubermatic/pull/14564))
+- KubeLB: the configuration in seed `seed.spec.kubelb.enableForAllDatacenters` can be used to allow kubeLB installation for all the datacenters belonging to the seed ([#14558](https://github.com/kubermatic/kubermatic/pull/14558))
```

Comment:
Update as follows
`- KubeLB: The configuration in the Seed `seed.spec.kubelb.enableForAllDatacenters` can be used to allow KubeLB installation for all the datacenters belonging to the Seed ([#14558](https://github.com/kubermatic/kubermatic/pull/14558))`